python_modules/dagster/dagster/_core/execution/run_metrics_thread.py

```diff
@@ -211,7 +223,7 @@ def _capture_metrics(
 
         except:
             if logger:
-                logger.error("Exception during capture of metrics, will cease capturing")
+                logger.exception("Exception caught during capture of metrics, will cease capturing")
```

Comment:
logger.warning("...", exc_info=True) is another option if we want it show up as a warning but not an error but sitl have the stack trace